client-app/src/common/plugins/store.info.plugin.ts

```diff
@@ -0,0 +1,17 @@
+import _Vue from "vue";
+
+/**
+ * Extend Vue with $storeInfo global property with such interface { baseUrl, storeName, currentLng }
+ */
+export function StoreInfoPlugin(Vue: typeof _Vue, options?: any){
+  const baseUrl = window.BASE_URL.replace(new RegExp("[/]+$"), "");
+  const storeName = (window as any).STORE_NAME;
+  const currentLng = (window as any).STORE_LNG;
+  Vue.$storeInfo = { baseUrl, storeName, currentLng };
+}
+
```

Comment:
It's better to have only single class/interface/function per file. Sometime we violate this rule, but I think IStoreInfo should be moved to another file

Reply:
I aggree